test/k8sT/Nightly.go

```diff
@@ -433,12 +445,11 @@ var _ = Describe("NightlyExamples", func() {
 		for _, image := range helpers.NightlyStableUpgradesFrom {
 			func(image string) {
 				It(fmt.Sprintf("Update Cilium from %s to master", image), func() {
-					helpers.InstallExampleCilium(kubectl, image)
+					By("Cleaning up everything before testing an upgrade")
+					cleanupPreTest(image)
 
-					err := kubectl.CiliumEndpointWaitReady()
-					Expect(err).To(BeNil(), "Endpoints are not ready after timeout")
 					var assertUpgradeSuccessful func()
-					assertUpgradeSuccessful, cleanupCallback = ValidateCiliumUpgrades(kubectl)
+					assertUpgradeSuccessful, cleanupCallback = ValidateCiliumUpgrades(kubectl, image, "k8s1:5000/cilium/cilium-dev:latest")
```

Comment:
`k8s1:5000/cilium/cilium-dev:latest` should be a const in helpers.

Reply:
Done